src/handles/handleDiagnostic.ts

```diff
@@ -267,12 +269,12 @@ async function handleLinter (
         range: {
           start: {
             // line and character is base zero so need -1
-            line: sumNum(line, -1, offsetLine),
-            character: sumNum(column, -1, offsetColumn)
+            line: sumNum(line, -1, offsetLineStart),
+            character: sumNum(column, -1, offsetColumnStart)
           },
           end: {
-            line: sumNum(endLine, -1, offsetLine),
-            character: sumNum(endColumn, -1, offsetColumn)
+            line: sumNum(endLine, -1, offsetLineEnd),
+            character: sumNum(endColumn, -1, offsetColumnEnd)
           }
```

Comment:
Change to

```
            line: sumNum(endLine, -1, offsetLineEnd || offsetLine),
            character: sumNum(endColumn, -1, offsetColumnEnd || offsetLine)
```

so it will not breaking the exists user's config.